components/butterfly/src/member.rs

```diff
@@ -326,22 +325,22 @@ impl FromProto<newscast::Rumor> for Membership {
     }
 }
 
+mod member_list {
+    #[derive(Clone, Debug)]
+    pub struct Entry {
+        pub member: super::Member,
+        pub health: super::Health,
+        pub health_updated_at: super::SteadyTime,
```

Comment:
Is there a benefit to storing Member and Health separately, as opposed to a Membership directly?

It essentially works out to the same thing, type-wise and space-wise, but might make it easier to talk about what exactly this `Entry` is... it records membership transitions.